src/3p-frame.js

```diff
@@ -212,22 +212,30 @@ export function resetBootstrapBaseUrlForTesting(win) {
  * @return {string}
  */
 export function getDefaultBootstrapBaseUrl(parentWindow, opt_srcFileBasename) {
-  if (parentWindow.defaultBootstrapBaseUrl) {
-    return parentWindow.defaultBootstrapBaseUrl;
-  }
   const srcFileBasename = opt_srcFileBasename || 'frame';
+  // Cached at filename level to ensure different url is returned.
+  parentWindow.defaultBootstrapBaseUrl =
```

Comment:
nit: can we rename `defaultBootstrapBaseUrl` to indicate it is now a map. `defaultBootstrapBaseUrlMap` or `defaultBootstrapBaseUrls`?